source/externals/ptl/cmake/Modules/Documentation.cmake

```diff
@@ -193,7 +193,7 @@ ENDMACRO(GENERATE_DOCUMENTATION)
 
 #------------------------------------------------------------------------------#
 # Macro to generate PDF manual from LaTeX using pdflatex
-# assumes manual is in ${CMAKE_SOURCE_DIR}/doc
+# assumes manual is in ${Geant4_SOURCE_DIR}/doc
```

Comment:
Please change back - we import PTL as an external and we try to limit changes as far as possible